src/models/list.js

```diff
@@ -0,0 +1,22 @@
+import mongoose from 'mongoose'
+
+const listTodoSchema = mongoose.Schema({
+    name: { type: String, required: true },
+    description: { type: String },
+    createdBy: {
+        type: mongoose.Schema.Types.ObjectId,
+        ref: 'User'
+    },
+    // A list has many todos
+    todos: [
```

Comment:
You would need to restructure your model a little bit. Child and parent must not reference each other. One point is enough